.github/workflows/ci.yml

```diff
@@ -473,6 +473,14 @@ jobs:
           TEST_RUNTIME: ${{ matrix.runtime }}
         run: |
           BDIR="$(mktemp -d -p $PWD)"
+
+          function cleanup() {
```

Comment:
Wondering if an `if: always()` would've worked for these (I see we're already using that for some steps below), or is the containerd process already killed by github actions itself on failure?

Reply:
Yeah. Checked the doc and we can have cleanup in the following step with always. Let's handle it in next pr👍